dashboard/app/views/levels/_free_response.html.haml

```diff
@@ -1,16 +1,33 @@
-:javascript
-  function getResult() {
-    return {
-      response: $('.response').val(),
-      result: true
-    };
-  }
+- level ||= @level
+- last_attempt ||= @last_attempt
+- index ||= nil
+- number ||= nil
+
+- if index
+  :javascript
+    window['level_#{index}'] = {
```

Comment:
Ideally the partials would follow a standard format (maybe a hidden form with `id='#{level.id}'`) and the `_level_group` partial would own aggregating the results?